modules/ROOT/nav.adoc

```diff
@@ -8,22 +8,23 @@
 * xref:external-identity.adoc[About Identity Management]
  ** xref:sso-prerequisites-about.adoc[About SSO Prerequisites and Limitations]
  ** xref:external-identity-index.adoc[External Identity Configurations]
-  *** xref:managing-users.adoc[To Configure SAML]
+  *** xref:managing-users.adoc[Configure SAML]
    **** xref:verify-saml-info-task.adoc[To Verify SAML Information]
```

Comment:
Remove leading “To” to match the other bullets?

Reply:
Yes!